libs/chatbot/src/lib/ui/styles.less

```diff
@@ -34,14 +34,13 @@
 .msla-chatbot-header-title-container {
   .msla-chatbot-header-title {
     font-weight: 600;
-    font-size: 24px;
+    font-size: 16px;
     line-height: 28px;
     margin: 0px 12px;
   }
 
   .msla-chatbot-header-subtitle {
     margin-left: 15px;
```

Comment:
very nit, but could you decrease the the margin-left of the subtitle so that it aligns  a bit better with the title

Reply:
done!